src/fundus/publishers/tr/haberturk.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+
+class HaberturkParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("article[property='articleBody']")
```

Comment:
Fundus separates articles into paragraphs and subheadlines introduced with a short summary. I don't speak Turkish, but from the layout it looks like this publisher supports all of them. Could you add the missing selectors and extract the content paragraph wise and not as s single block.

Reply:
I have chosen it as a subheading, I could also adopt it as a summary. Do you think it would be more suitable for a summary?

Reply:
@ozelalisen Yes, that part pretty much summarizes the rest of the article ;)

Reply:
Alright, that is fixed